spec/jobs/open_source_slack_job_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe OpenSourceSlackJob do
+  describe '#perform' do
+    it 'calls SlackService' do
+      expect(SlackService)
+        .to receive(:open_source_reminder)
+        .once
```

Comment:
would it be better to test that there was a request call made to slack (using WebMock)?